openreview/venue_request/process/deployProcess.py

```diff
@@ -327,3 +329,68 @@ def process(client, note, invitation):
         },
         signatures=['~Super_User1']
     ))
+
+    if (forum.content.get('Submission Deadline') and forum.content.get('Paper Matching')):
+        activation_date = forum.content.get('Submission Deadline').strip()
+        try:
+            activation_date = datetime.strptime(activation_date, '%Y/%m/%d %H:%M')
+        except ValueError:
+            activation_date = datetime.strptime(activation_date, '%Y/%m/%d')
+        matching_invitation = openreview.Invitation(
+            id = SUPPORT_GROUP + '/-/Request' + str(forum.number) + '/Matching_Stage',
+            super = SUPPORT_GROUP + '/-/Matching_Stage',
+            invitees = readers,
+            cdate = openreview.tools.datetime_millis(activation_date),
+            reply = {
+                'forum': forum.id,
+                'replyto': forum.id,
+                'readers' : {
+                    'description': 'The users who will be allowed to read the above content.',
+                    'values' : readers
+                },
+                'writers': {
+                    'values':[],
+                },
+                'content': {
+                    'title': {
+                        'value': 'Paper Matching Setup',
+                        'required': True,
+                        'order': 1
+                    },
+                    'matching_group': {
+                        'description': 'Please select the group you want to set up matching for.',
+                        'value-dropdown' : [conference.get_id() + '/Reviewers', conference.get_id() + '/Area_Chairs'] if forum.content.get('Area Chairs (Metareviewers)') == "Yes, our venue has Area Chairs" else [conference.get_id() + '/Reviewers'],
+                        'required': True,
+                        'order': 2
+                    },
+                    'build_conflicts': {
```

Comment:
```suggestion
                    'compute_conflicts': {
```

I would like to call it compute instead of build, it is more self descriptive